cloudformation/policies/parallelcluster-policies.yaml

```diff
@@ -448,6 +448,7 @@ Resources:
               - cloudwatch:PutMetricAlarm
               - cloudwatch:DeleteAlarms
               - cloudwatch:DescribeAlarms
+              - cloudwatch:PutCompositeAlarm
```

Comment:
We should fix the `tests/iam_policies/user-role.cfn.yaml` too

Reply:
This was already added by Giacomo in https://github.com/aws/aws-parallelcluster/commit/9aa5c5f1e2f227caef96fd7664bc7ad005bcec3e